uktrade/data-hub-frontend
# CLS2-675: add similar task

## Description of change

This PR allows the users to create a task starting from a pre existing one. The new task should not have `title` `description` and `id` so that a new one gets created.

[Jira ticket](https://uktrade.atlassian.net/browse/CLS2-675)

## Test instructions

- Create a new task
- In the task details page click on create similar task
- Fill in the title field
- The task list page should now show the new task alongside the existing one

## Screenshots

### After
![Screenshot 2024-01-11 at 15 14 00](https://github.com/uktrade/data-hub-frontend/assets/1381563/97c1ae32-dfa8-4940-87e6-91b668371fd9)
![Screenshot 2024-01-11 at 15 14 11](https://github.com/uktrade/data-hub-frontend/assets/1381563/56d69481-9d5a-4548-8958-34f436f1f733)
![Screenshot 2024-01-11 at 15 14 22](https://github.com/uktrade/data-hub-frontend/assets/1381563/25c5148b-4404-4bf1-879e-b8b8193b9dff)
![Screenshot 2024-01-11 at 15 14 29](https://github.com/uktrade/data-hub-frontend/assets/1381563/dcad142f-601f-445b-b680-3ac6eb9b2d91)

_Add a screenshot_

## Checklist
